tune_sklearn/tune_basesearch.py

```diff
@@ -209,7 +209,7 @@ def __init__(self,
         if early_stopping and self._can_early_stop():
             self.max_iters = max_iters
             if early_stopping is True:
-                early_stopping = "AsyncHyperBandScheduler"
+                self.early_stopping = "AsyncHyperBandScheduler"
             elif isinstance(early_stopping, str):
```

Comment:
I think this doesn't work; 

the idea here was to make early_stopping = "AsyncHyperBandScheduer" so it is checked in L214.

Reply:
otherwise, self.early_stopping is never actually set into a scheduler object

Reply:
Whoops yeah you're right. Changed to address this.